### PR TITLE
build: add teamcity scripts

### PR DESCRIPTION
**Pull request by jordanlewis**

### CODE REVIEW

**Review by petermattis — APPROVED** [review]
LGTM though perhaps @tamird or @benesch should put eyes on it.

File: Makefile [hunk 1]
```diff
@@ -72,4 +77,4 @@ check:
 	@echo "gofmt (simplify)"
 	@! gofmt -s -d -l . 2>&1 | grep -vE '^\.git/'
 	@echo "goimports"
-	@! goimports -l . | grep -vF 'No Exceptions'
+	@! goimports -l . | grep -vF 'No Exceptions'k
```
Comment (benesch) [inline Makefile]: Stray `k`.

File: Makefile [hunk 2]
```diff
@@ -47,6 +48,10 @@ kv:
 ycsb:
 	$(GO) build -tags '$(TAGS)' $(GOFLAGS) -ldflags '$(LDFLAGS)' -v -i -o ycsb/ycsb ./ycsb
 
+.PHONY: tpcc
+tpcc:
+	$(GO) build -tags '$(TAGS)' $(GOFLAGS) -ldflags '$(LDFLAGS)' -v -i -o tpch/tpcc ./tpcc
```
Comment (benesch) [inline Makefile]: `tpch/tpcc` looks wrong. `tpcc/tpcc`?

Also, it's probably time to merge these rules:

```make
.PHONY: kv ycsb tpcc tpch
kv ycsb tpcc tpch:
$(GO) build -tags '$(TAGS)' $(GOFLAGS) -ldflags '$(LDFLAGS)' -v -i -o $@/$@ $@
```

File: Makefile [hunk 3]
```diff
@@ -34,6 +34,7 @@ test:
 
 .PHONY: deps
 deps:
+	$(GO) get -u github.com/kisielk/errcheck
 	$(GO) get -d -t ./...
 
 .PHONY: build
```
Comment (benesch) [inline Makefile:40]: `build` should depend on `tpcc` and `tpch` too, no?

**Comment by benesch** [comment]
Also looks like `make check` has legitimate failures 😞.

**Comment by jordanlewis** [comment]
TFTRs, acted on all suggestions from @benesch.

**Pull request merged**